### DISCUSSION TITLE
ModuleNotFoundError: No module named 'pyshark'

### DISCUSSION THREAD

**Discussion by atzelepi** [KimiNewt/pyshark#601] [Q&A]
![error](https://user-images.githubusercontent.com/64417556/193087517-25900905-e292-4250-99ef-8a4260062889.png)
just imported pyshark nad have the error   ModuleNotFoundError: No module named 'pyshark'

**Comment by Dhyey-Thummar** [comment]
Yeah, I also faced the same issue while using `pip install` . Try using the different method mentioned i.e. cloning the repo and running the setup file :
`git clone https://github.com/KimiNewt/pyshark.git`
`cd pyshark/src`
`python setup.py install`

**Reply by securitygoodie** [reply]
I tried to install and got pyshark installed this way. But still getting same error.
Anyone having luck to install pyshark and use it with python?
